dot/rpc/modules/author.go

```diff
@@ -40,7 +66,9 @@ type AuthorModule struct {
 type KeyInsertRequest []string
 
 // Extrinsic represents a hex-encoded extrinsic
-type Extrinsic string
+type Extrinsic struct {
+	Data string
+}
```

Comment:
this changes the extrinsic format in JSON, is this related to the polkadot-js issues?

Reply:
This is needed for unmarshalling the JSON request. This was broken but it didn't error out as the test was skipped.